sdk/core/azure-core/src/main/java/com/azure/core/spatial/GeometryPosition.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.spatial;
+
+/**
+ * Represents a geometric position.
+ */
+public final class GeometryPosition {
+    private final double longitude;
+    private final double latitude;
+
+    private final Double altitude;
```

Comment:
Search use-case does not use altitude. So for serialization, this field is not required. Make sure the serializer will remove the field once it is null. Or make it optional.

Reply:
Since it is a `Double` it is optional, if it is set it will be included in the serialization. I don't know what the Search service will do if it is included though.

Reply:
There is a slightly chance to go wrong.
If sdk set the JsonInclude (JsonInclude(Include.ALWAYS)), then the serializer will include the field set to null. 

Also, I am curious how would it be like for serialization. 
Here is what search looks like:
```
"type":"Point","coordinates":[-122.131577,47.678581]
```

Reply:
We'd better to assume service will throw error if you provide something unwanted in core spatial.

Reply:
To make my point clear, if we set 'altitude' anyway in constructor, in some cases, we send out altitude even it sets to null.  (e.g. use JsonInclude(Include.ALWAYS)). So it is better to remove final and have setter to set the optional field.

Reply:
These classes will be using a custom serializer and deserializer handler in Azure Core which won't include altitude if it is `null`. I've added them in the last commit.

Reply:
Just saw the commit. That solves the problem. Thanks!